applications/salk-portal/src/components/ExperimentSidebar.jsx

```diff
@@ -125,6 +174,16 @@ const ExperimentSidebar = () => {
   };
 
   const sidebarClass = `${classes.sidebar} scrollbar ${shrink ? `${classes.shrink}` : ``}`;
+  const populationColours = ['green', 'sky', 'purple', 'brown', 'blue'];
+
+  const PopulationLabel = ({labelText}) => {
+    return (
+      <div className='sidebar-label'>
```

Comment:
@paularah For a label the best component is use Typography from material-ui.
Instead of using multiple divs to display text its better to use Material components like Typography, and may be for bullets/dots typography with a component span or so.